sklearn/manifold/_mds.py

```diff
@@ -335,6 +335,16 @@ class MDS(BaseEstimator):
         The final value of the stress (sum of squared distance of the
         disparities and the distances for all constrained points).
 
+    dissimilarity_matrix_ : ndarray, shape (n_samples, n_samples)
```

Comment:
```suggestion
    dissimilarity_matrix_ : ndarray of shape (n_samples, n_samples)
```

Reply:
In `_mds.py` all descriptions use `ndarray, shape()`. So I think I'll keep the original. 
All other changes will be made in the new submit.

Reply:
Please make the change to follow our documentation guidelines. We might have a PR already making the `_mds` module following our guidelines. In this case, your changes will not be shown on this PR and will not be corrected. If you really want to make things a bit more consistent you can edit the section `Attributes` of the documentation of the estimator where you add the new attribute.

Reply:
You are right I didn't think of it. Thanks for correcting them.